Makefile

```diff
@@ -495,8 +493,6 @@ help:
 	@echo 'to check the official mods for erroneous yaml files, run:'
 	@echo '  make test'
 	@echo
-	@echo 'to generate documentation aimed at modders, run:'
-	@echo '  make docs'
 	@echo
```

Comment:
Can you please remove one of these empty lines too? Otherwise there is two between 'make test' and 'to install, run'.